### DISCUSSION TITLE
Is a negotiated rate ever based just on an ICD code

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/price-transparency-guide#68] [Q&A] [answered]
In the in-network-rates file description, the example for billing-code-type-version is the version of the ICD code.
Is an ICD code ever used as the exclusive billing code for a negotiated rate?

My understanding was that claims would include an ICD code to document the medical condition being treated, but that the items being billed were described by other coding systems e.g. CPTs, HCPCS, MS-DRG.

If this is the case, then this is a misleading example of a billing-code-type-version and as discussed in other posts there is a need for multiple billing codes for a single (non-bundled) item.

**Comment by ghost** [comment]
Correct, I was just on a call today with my business SMEs and they told me the same thing when I mentioned ICD codes were listed. I would be interested if anyone also knows differently.

**Reply by tdfow** [reply]
the bundled example is where they have it used, where the bundled codes are based on CPT codes.  Not sure why they are using a ICD 9 example, as those are no longer applicable.  Maybe the only applicable use is a bundled scenario?

**Reply by nmayle** [reply]
Do bundles sometimes get billed by the ICD code?  Do the payers keep the ICD code associated with a procedure (or bundle) in their database of prices?

**Reply by shaselton-usds** [reply]
@tdfow @nmayle The examples are with dummy data and not based on a real life situation. The examples are meant to show how a schema can be implemented.

**Comment by shaselton-usds** [comment, marked as answer]
@nmayle I don't know if you've had the chance to see the discussion happening here https://github.com/CMSgov/price-transparency-guide/discussions/9#discussioncomment-783535

It appears that ICD is used in contracting/negotiated rates. I'll go ahead and close this thread if you're good with it.
